docs-source/content/userguide/istio/istio.md

```diff
@@ -6,32 +6,31 @@ weight: 1
 
 #### Overview
 
-WebLogic Server Kubernetes Operator version 2.3 and later includes experimental support for Istio 1.2.2 and later.
+WebLogic Server Kubernetes Operator version 2.3 and later includes support for Istio 1.2.2 and later.
 This support allows you to run the operator itself, and WebLogic domains managed by
 the operator, with Istio sidecar injection enabled.  It will allow you to use
 Istio gateways and virtual services to access applications deployed in these domains.
 If your applications have suitable tracing code in them, then you will also be able to
 use distributed tracing, such as Jaeger, to trace requests across domains and to
 other components and services that have tracing enabled.
 
+You can learn more about Istio at [Istio](https://istio.io/latest/docs/concepts/what-is-istio/)
+
 #### Limitations
 
-The current experimental support for Istio has these limitations:
+The current support for Istio has these limitations:
 
-* It is tested with Istio 1.2.2 and later (up to 1.5), however it is tested with both single and
+* It is tested with Istio 1.2.2 and later (up to 1.5.4), however it is tested with both single and
```

Comment:
Just to clarify, what Istio versions are we testing?

Reply:
Pani have tested 1.4.2 and also 1.5.4.  I am using 1.6.0 and latest 1.6.1 but I didn't do all the QA testings.